docs/en/tutorials/book-store/part-03.md

```diff
@@ -30,16 +30,16 @@ In this tutorial series, you will build an ABP based web application named `Acme
 
```

Comment:
https://github.com/EmreKendirli/abp/blob/docs-unification-1/docs/en/tutorials/book-store/part-03.md#deleting-a-book-1
<img width="825" alt="image" src="https://github.com/abpframework/abp/assets/16813853/6c4be9e0-a096-44c8-b014-9ebe6a6a5545">

The link is incorrect